### ISSUE TITLE
file name of default kraken configuration in util.sh must be changed.

### ISSUE THREAD

**Issue by keyolk** [samsung-cnct/k2#72]
from the below.
```
bin/util.sh
```

default configuration generated by the command "up.sh --generate" 
is config.yaml
```
function generate_config {
  inf "Generating config file at: $1"
  mkdir -p "${1%/*}"
  cp "${KRAKEN_ROOT}/ansible/roles/kraken.config/files/config.yaml" "${1}"
  exit 0
}
```

but the variable KRAKEN_CONFIG is set with suffix .yml
```
if [ -z ${KRAKEN_CONFIG+x} ]; then
  warn "--config not specified. Using ~/.kraken/config.yml as location"
  KRAKEN_CONFIG="~/.kraken/config.yml"
fi
```

it cause minor problem.

**Comment by maratoid** [comment]
https://github.com/samsung-cnct/k2/pull/73

**Comment by maratoid** [comment]
fixed via #73
